iothub/service/src/IotHubConnection.cs

```diff
@@ -428,17 +422,9 @@ private async void OnRefreshToken()
             X509Chain chain,
             SslPolicyErrors sslPolicyErrors)
         {
-            if (sslPolicyErrors == SslPolicyErrors.None)
-            {
-                return true;
-            }
-
-            if (s_disableServerCertificateValidation.Value && sslPolicyErrors == SslPolicyErrors.RemoteCertificateNameMismatch)
-            {
-                return true;
-            }
-
-            return false;
+            return sslPolicyErrors == SslPolicyErrors.None 
+                || (s_disableServerCertificateValidation.Value 
```

Comment:
If `s_disableServerCertificationValidation` is a bool, can we rename it to something that sounds more like a bool?

s_shouldDisableServerCertificationValidation?

Reply:
sure !